data/names.txt

```diff
@@ -1073,6 +1126,124 @@ phrase "standalone civilian ship names"
 		"Righteous"
 		"Venerable"
 		"Mjolnir"
+		"Starscreamer"
+		"Barrabas"
+		"Musashi"
+		"Misaka"
+		"Belo Horizonte"
+		"Visby"
+		"Pyotr Velikiy"
+		"Invincible"
+		"Renowned"
+		"Courageous"
+		"Indomitable"
+		"Endurance"
+		"Avenger"
+		"Unrivaled"
+		"Retribution"
+		"Allegiance"
+		"The Mountain and I"
+		"Washed Away"
+		"Cash is King"
+		"Equality"
+		"Equanimity"
+		"Eudamonia"
+		"Independence"
+		"Interdependence"
+		"Mutuality"
+		"Proteus"
+		"William Rockefeller"
+		"Lewis and Clark"
+		"Leif Erikson"
+		"Marco Polo"
+		"Zheng He"
+		"Vasco de Gama"
+		"Amerigo Vespucci"
+		"Jacques Cartier"
+		"James Cook"
+		"Horatio Hornblower"
+		"Thorstein Veblen"
+		"Karl Marx"
+		"John Maynard Keynes"
+		"Milton Friedman"
+		"Missouri"
+		"Montana"
+		"Essex"
+		"Karaboudjan"
+		"Immer Essen"
+		"Scotia"
+		"Sherwood"
+		"Thunderfish"
+		"Venture"
+		"Arabella"
+		"Baalbek"
+		"Barracuda"
+		"Bellipotent"
+		"Calypso"
+		"Forrestal"
+		"Compass Rose"
+		"Covenant"
+		"Dazzler"
+		"Fenton"
+		"Keeling"
+		"Marie Celeste"
+		"Hodgson"
+		"Okinawa"
+		"Orcus"
+		"Pharaoh"
+		"Pyramus"
+		"Reluctant"
+		"Saltash"
+		"Sturgeon"
+		"Swordfish"
+		"Seaview"
+		"Starview"
+		"Umbriago"
+		"Valparaiso"
+		"Kandahar"
+		"Colonia"
+		"Shenandoah"
+		"Destiny"
+		"Rockingham"
+		"Atropos"
+		"Hotspur"
+		"Speedwell"
+		"Sutherland"
+		"Magicienne"
+		"Papillion"
+		"Vestal"
+		"Remembrance"
+		"Ark"
+		"Sardine"
+		"Pinafore"
+		"Minnow"
+		"Skydiver"
+		"Vondel"
+		"Elisabeth Dane"
+		"Valkyrie"
+		"Geofon"
+		"Beowulf"
+		"Nunki"
+		"Mercury"
+		"Venus"
+		"Mars"
+		"Jupiter"
+		"Saturn"
+		"Neptune"
+		"Uranus"
```

Comment:
seems you purposefully organized them based on distance from the sun, which means Neptune should be below this one, not above it
(although I remember there was a planet whose trajectory sometimes brings it close to the sun than another planet)

Reply:
lol oops

Reply:
Pluto moves inside Neptune's orbit for a short part of the plutonian year.

Reply:
Sadly, all planetary orbits in Endless Sky are circular.

Reply:
Pluto is not a planet, however.
*leans back and grabs popcorn*